config.yaml

```diff
@@ -0,0 +1,66 @@
+publisher:
+  name: ShapeShift
+  address: ''
+  website: https://shapeshift.com
+  email: legal@shapeshift.io
+  media:
+    - purpose: icon
+      uri: ./src/static/icon-512.png
+app:
+  name: ShapeShift
+  address: ''
+  android_package: com.shapeshift.droid_shapeshift
+  urls:
+    license_url: https://shapeshift.com/privacy-policy
+    copyright_url: https://shapeshift.com/terms-of-service
+    privacy_policy_url: https://shapeshift.com/privacy-policy
+    website: https://shapeshift.com
+  media:
+    - purpose: icon
+      uri: ./src/static/icon-512.png
+release:
+  address: ''
+  media:
+    - purpose: icon
+      uri: ./src/static/icon-512.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/1.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/2.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/3.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/4.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/5.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/6.png
+    - purpose: screenshot
+      uri: ./solanaStoreMedia/7.png
+  files:
+    - purpose: install
+      uri: ./solanaStoreMedia/shapeshift-v3.2.0-signed.apk
```

Comment:
Is this a manually updated config file we'll need to change every build?

Reply:
We will need to change it on every build unfortunately, as long as we are not changing native file, no need for a build, but if we are, we will need to update it